app/assets/javascripts/controllers/CommonChatController.js

```diff
@@ -274,8 +287,10 @@ export default class CommonChatController {
 
     onConfirmEndChat() {
         this._moveToClosingState();
-        this._sendPostChatSurveyWebchat(this.sdk).beginPostChatSurvey(webchatSurvey, automaton, timestamp);
-        this.container.showPage(new PostChatSurveyWebchat((page) => this.onPostChatSurveyWebchatSubmitted(page)));
+        //this._sendPostChatSurveyWebchat(this.sdk).beginPostChatSurvey(webchatSurvey, automaton, timestamp)
```

Comment:
Can we get rid of commented code?

Reply:
No, will make DEP-2480 easier as we need both methods for that ticket